app/src/main/java/org/mozilla/focus/utils/Settings.kt

```diff
@@ -306,6 +298,53 @@ class Settings(
         false
     )
 
+    fun getSitePermissionsSettingsRules() = SitePermissionsRules(
+        notification = SitePermissionsRules.Action.BLOCKED,
+        microphone = SitePermissionsRules.Action.BLOCKED,
+        location = SitePermissionsRules.Action.BLOCKED,
+        camera = SitePermissionsRules.Action.BLOCKED,
+        autoplayAudible = getAutoplayRules().first,
+        autoplayInaudible = getAutoplayRules().second,
+        persistentStorage = SitePermissionsRules.Action.BLOCKED,
+        mediaKeySystemAccess = SitePermissionsRules.Action.BLOCKED
+    )
+
+    fun reloadCurrentTab() {
+        context.components.sessionUseCases.reload.invoke(context.components.store.state.selectedTabId)
+    }
```

Comment:
I think we could use the `sessionUseCases` directly where we are calling this function, instead of adding it as part of settings :) .